### DISCUSSION TITLE
tried some key mappings but failed

### DISCUSSION THREAD

**Discussion by DanielAjoy** [h3rald/min#162] [General]
I tried some key mappings but failed. I tried:

```
(goToStart) 'ctrl+a mapkey

and 

(home) 'ctrl+a mapkey
```

The reference says

> At present, only the key names and sequences defined in the nimline library are supported.

But I couldn't figure out what *sequences* were available or how to use them

**Comment by h3rald** [comment]
Well... you were nearly there! Essentially you can bind a key sequence like ctrl+<key> or similar to any quotation, like this:

```
(quit) 'ctrl+q mapkey
```

...then pressing CTRL+q will execute `quit` and exit the min REPL.
